.github/workflows/lint.yml

```
@@ -25,5 +25,5 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v1
-      - run: pip install 'cpplint==1.5.4'
+      - run: pip install cpplint
       - run: cpplint src/*.h src/*.cc
```

Come to think of it, I also recommend changing this line so that we don't have to manually keep it in sync with `etc/cpplint.sh`. In particular, your `libsemigroups-v2` branch changes `etc/cpplint.sh`, and so if you make this change now, you'll be future-proofing the listing CI against that change.
```suggestion
      - run: bash etc/cpplint.sh
```

---

Good idea!